src/Sylius/Bundle/ApiBundle/Controller/UploadTaxonImageAction.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Controller;
+
+use Sylius\Bundle\ApiBundle\Creator\TaxonImageCreatorInterface;
+use Sylius\Component\Core\Model\TaxonImageInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/** @experimental */
+final class UploadTaxonImageAction
+{
+    public function __construct(private TaxonImageCreatorInterface $taxonImageCreator)
+    {
+    }
+
+    public function __invoke(Request $request): TaxonImageInterface
+    {
+        return $this->taxonImageCreator->create(
+            $request->attributes->get('code'),
```

Comment:
```suggestion
            $request->attributes->get('code', ''),
```
or
```suggestion
            $request->attributes->getString('code'),
```
if the symfony version allows that;
I know that the default routing requires `code` to be present, but better safe than sorry.

Reply:
+1 for `getString`

Reply:
`getString` method would be great here, but is does not exist in SF 5.4

Reply:
Go with the first suggestion then 💃